### DISCUSSION TITLE
How do I know if a node is connected with another?

### DISCUSSION THREAD

**Discussion by JazielIsai** [xyflow/xyflow#2243] [Q&A]
I want to know if a specific node has connections?

thank you for your help.

**Comment by bcakmakoglu** [comment]
You can use the `getConnectedEdges` utility to find edges that are connected to a node and from there you can find out what other node's are connected to those edges (by checking the edges' `source` and `target` properties).
[Here are the docs.](https://reactflow.dev/docs/api/graph-util-functions/#getconnectededges)

**Reply by JazielIsai** [reply]
Thank you very much
